apps/aefate/test/aefa_sophia_test.erl

```diff
@@ -62,6 +74,7 @@ make_call(Contract, Function, Arguments) ->
     Calldata = {tuple, {Function, {tuple, EncArgs}}},
     #{ contract => pad_contract_name(Contract),
        gas      => 1000000,
+       value    => 10000,
        call     => aeb_fate_encoding:serialize(Calldata) }.
 
```

Comment:
Might consider adding a `Call.value` to the remote test. Perhaps
```
  function remote(x : int) = x * (2 + Call.value)
```

Reply:
I would if the compiler could handle it ;-)
I see another PR in the future.

Reply:
🔮